### PR TITLE
Update RelativeHumidityMeasurement in Matter Linux Air Quality Example README.md

### PR DESCRIPTION
**Pull request by lboue**
Fix [Matter Linux Air Quality Example README.md](https://github.com/project-chip/connectedhomeip/tree/master/examples/air-quality-sensor-app/linux). The value is expressed in hundredths of a percentage.
So change it to 60% instead of 0,6%.

### CODE REVIEW

File: examples/air-quality-sensor-app/linux/README.md [hunk 1]
```diff
@@ -136,10 +136,11 @@ $ echo '{"Name":"TemperatureMeasurement","NewValue":1800}' > /tmp/chip_air_quali
 
 ### Trigger Humidity change event
 
-Generate event `RelativeHumidityMeasurement`, to change the temperate value.
+Generate event `RelativeHumidityMeasurement`, to change the temperate value
```
Comment (bzbarsky-apple) [inline examples/air-quality-sensor-app/linux/README.md:139]: Pre-existing, but what is "the temperate value" and how is it changed by generating a RelativeHumidityMeasurement event?

Comment (lboue) [reply examples/air-quality-sensor-app/linux/README.md]: Fixed by [1f5d07c](https://github.com/project-chip/connectedhomeip/pull/32254/commits/1f5d07c6662bb203d7bfec2123af830f4ef55268)

**Review by lboue — COMMENTED** [review]
Fixed

**Comment by lboue** [comment]
Can you validate this PR?

**Review by bzbarsky-apple — APPROVED** [review]

**Comment by lboue** [comment]
Can you merge this PR?

**Review by yufengwangca — APPROVED** [review]

**Pull request merged**